_integration-schemas/ms-teams/v1/foreign-keys.md

```diff
@@ -0,0 +1,153 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas-foreign-keys/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+tap-reference: "integration"
+
+version: ""
```

Comment:
I'm gonna fix this because I want to do a few things in here, but remember to double-check that these two guys are correct or all the FK work you do won't show up